lib/assets/javascripts/twine.coffee

```diff
@@ -0,0 +1,349 @@
+((root, factory) ->
+  if typeof root.define == 'function' && root.define.amd
+    root.define
+  else if typeof module == 'object' && module.exports
+    module.exports = factory()
+  else
+    root.Twine = factory()
+) this, ->
```

Comment:
Can you wrap the arguments in parens? I find it hard to read this in CoffeeScript